src/SamplesApp/UITests.Shared/Windows_ApplicationModel/ClipboardTests.xaml.cs

```diff
@@ -45,7 +45,7 @@ internal class ClipboardTestsViewModel : ViewModelBase
 		private bool _isObservingContentChanged = false;
 		private string _lastContentChangedDate = "";
 		private string _text = "";
-		private BitmapImage _bmp;
+		private BitmapSource _bmp;
```

Comment:
This particular sample is not marked as manual tests, let's make it so and let's also add steps to validate what we're adding support for in this PR.